src/shared/components/palette/case-flag/components/add-comments/add-comments.component.ts

```diff
@@ -2,7 +2,7 @@ import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { FormControl, FormGroup } from '@angular/forms';
```

Comment:
@johnbenjamin-hmcts - I think you can also remove `EventEmitter` and `Output` from the imports, given the component no longer emits anything

Reply:
Thank you @Daniel-Lam - I used the command Shift+Alt+O to correct the imports :)